packages/block-editor/src/components/alignment-control/ui.js

```diff
@@ -32,7 +32,7 @@ function AlignmentUI( {
 	onChange,
 	alignmentControls = DEFAULT_ALIGNMENT_CONTROLS,
 	label = __( 'Align text' ),
-	describedBy = __( 'Change text alignment' ),
```

Comment:
This is probably a breaking change for this component. The easy way to avoid the braking change would be to keep the `describedBy` prop, and internally pass it to the `description` prop in `toggleProps`.

Alternatively, we'd probably need to do something similar to what we've done in `Button`, where we support the old version alongside the new version.

Reply:
@ciampo I understand the potential concern. However, I couldn't find any usage of the `describedBy` prop for the `AlignmentControl` component. The prop is not even documented in the [AlignmentControl readme](https://github.com/WordPress/gutenberg/blob/d1211ff7de85c66d45ad144f8c80323d875f964e/packages/block-editor/src/components/alignment-control/README.md). This component is used internally in the block-editor, is this change a real concern?
Rather, I'd say it's a good opportunity to _document_ it better. I discovered a few more issues and inconsistencies that I'll detail in a new comment.

Reply:
> This component is used internally in the block-editor

This change would affect the APIs of `AlignmentControl` and `AlignmentToolbar`, which are both public APIs of the `@wordpress/block-editor` package.

However, given that:

- as you mentioned, the property is not documented;
- as you mentioned, we couldn't find any current usages of the prop;
- any user using the old `describedBy ` prop won't experience breaking, but only an unexpected description (which fallback to the default "Change text alignment")
- the change is an overall improvement and aligns with the changes in the `Button` component;

I agree with you that it's ok to just swap the prop name from `describedBy` to `describe`